### PR TITLE
chore: enable prettier for json and reformat

### PR DESCRIPTION
**Pull request by dominikg**
can we add this to blame ignore here or do we have to merge into main first?

### CODE REVIEW

**Comment by ghostdevv** [comment]
All you should need is a file called `.git-blame-ignore-revs` with the commit hash in `7300d3b37b2b8b4099fccc0ecc3f8e4fef1151e8`

File: .github/renovate.json5
Comment (bluwy) [inline .github/renovate.json5:1]: This json5 formatting is a bit weird, but if you're fine with it, I'm fine with it too 😄

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   "lint-staged": {
     "*.{js,ts,svelte,html,md,svx}": "eslint --cache --fix",
     "*.{css,scss,svelte,html,js,ts,svx,md}": [
-      "prettier --cache --write"
+      "prettier --cache --ignore-unknown --write"
     ]
```
Comment (bluwy) [inline package.json:58]: Should we expand this to json, json5, yml, mjs, etc? Or maybe a catch all `*`

**Comment by bluwy** [comment]
> can we add this to blame ignore here or do we have to merge into main first?

We'd need to merge in main first since the commit hash will change after merge (commits are derived from commit time too)

File: .github/renovate.json5
Comment (dominikg) [reply .github/renovate.json5:1]: good catch, moved it to ignore

File: package.json (hunk 1 above)
Comment (dominikg) [reply package.json]: thanks, * is correct together with --ignore-unknown.

**Pull request merged**